internal/app/service/user_login_service.go

```diff
@@ -0,0 +1,59 @@
+package service
+
+import (
+	"context"
+
+	"os"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+	"github.com/joshsoftware/profile_builder_backend_go/internal/pkg/errors"
+)
+
+type UserLoginServive interface {
+	GenerateLoginToken(ctx context.Context, email string) (string, error)
+}
+
+func (userService *service) GenerateLoginToken(ctx context.Context, email string) (string, error) {
+	if len((email)) == 0 {
```

Comment:
handle this in 
internal/api/handler/user_login_handler.go

Reply:
handled in handler layer